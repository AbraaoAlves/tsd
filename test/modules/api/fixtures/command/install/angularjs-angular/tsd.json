{
  "typingsPath": "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\command\\install\\angularjs-angular\\typings",
  "version": "v4",
  "repo": "borisyankov/DefinitelyTyped",
  "ref": "master",
  "installed": {
    "jquery/jquery.d.ts": {
      "commit": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "blob": "99969f710e2dbcac67382988edc7224550d29b86"
    },
    "angularjs/angular.d.ts": {
      "commit": "a8d8c2ac668fbb99cb60fb5d25c663e9356fee14",
      "blob": "c44f3408e4b90c2572045d231a91f6042cf2e8da"
    }
  }
}