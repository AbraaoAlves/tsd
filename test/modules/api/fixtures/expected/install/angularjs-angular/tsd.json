{
  "typingsPath": "D:\\_Editing\\github\\tsd\\tsd-origin\\test\\modules\\api\\tmp\\result\\install\\angularjs-angular\\typings",
  "version": "v4",
  "repo": "borisyankov/DefinitelyTyped",
  "ref": "master",
  "installed": {
    "angularjs/angular.d.ts": {
      "commit": "3bc329f49a971b3dae92ca682cb81816dbe31a53",
      "blob": "c44f3408e4b90c2572045d231a91f6042cf2e8da"
    },
    "jquery/jquery.d.ts": {
      "commit": "3bc329f49a971b3dae92ca682cb81816dbe31a53",
      "blob": "99969f710e2dbcac67382988edc7224550d29b86"
    }
  }
}